### DISCUSSION TITLE
[4.0] Custom Fields output for non-HTML formats

### DISCUSSION THREAD

**Discussion by mbabker** [joomla/joomla-cms#47196] [4. Rejected Features]
Use Case:

The security announcements use a standardized template for all posts with a standard set of fields.  Presumably we could build a set of fields for this category and make the articles use those rather easily.  However, this category also broadcasts a RSS feed and is used by Feedburner for email notifications.

Would it be practical/possible to use the custom field data in the standard com_content feed view to still build correct entries?

I know full well non-HTML output in Joomla is rather limited as a whole (lacking plugin hooks, layout use, etc.) so this may be opening a Pandora's Box.

**Comment by mbabker** [comment]
Hmm, nevermind.

**Comment by wilsonge** [comment]
Reopening as important for webservices in j4

**Comment by rdeutz** [comment]
The Joomla Maintainer Team decided on the 25.02.2026 meeting to reject this feature request because the API would be a better place to implement it.
